index.html

```diff
@@ -69,130 +69,7 @@ <h1>
-        </p>
-        <div class="languages">
-          <ul>
-            <li class="lang">HTML</li>
-            <li class="lang">CSS</li>
-            <li class="lang">JavaScript</li>
-          </ul>
-        </div>
-        <div class="btns">
-          <button class="button" id="BTN">See live<i class="fa fa-external-link"></i></button>
-        <button class="button" id="BTN">See source <i class="fa fa-github"></i></button>
-        </div>
-      </div>
-    </div>
-    <div class="works2 animate">
-      <img class="photo2" src="images/multipost project.png" />
-      <div class="part2">
-        <h1 class="title">
-          Multi-Post Stories
-        </h1>
-        <ul class="canopy">
-          <li class="can"><b>CANOPY</b></li>
-          <li class="dev"><b> Back End Dev</b></li>
-          <li class="dev"><b> 2015</b></li>
-        </ul>
-        <p class="presentation3">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </p>
-        <div class="languages">
-          <ul>
-            <li class="lang">HTML</li>
-            <li class="lang">CSS</li>
-            <li class="lang">JavaScript</li>
-          </ul>
-        </div>
-        <button class="button">See project</button>
-      </div>
-    </div>
-    <div class="works3 animate">
-      <img class="photo3" src="images/gym project.png" />
-      <div class="part3">
-        <h1 class="title">Tonic</h1>
-        <ul class="canopy">
-          <li class="can"><b>CANOPY</b></li>
-          <li class="dev"><b> Back End Dev</b></li>
-          <li class="dev"><b> 2015</b></li>
-        </ul>
-        <p class="presentation3">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </p>
-        <div class="languages">
-          <ul>
-            <li class="lang">HTML</li>
-            <li class="lang">CSS</li>
-            <li class="lang">JavaScript</li>
-          </ul>
-        </div>
-        <button class="button">See project</button>
-      </div>
-    </div>
-    <div class="works4 animate">
-      <img class="photo4" src="images/printing project.png" />
-      <div class="part4">
-        <h1 class="title">
-          Multi-Post Stories
-        </h1>
-        <ul class="canopy">
-          <li class="can"><b>CANOPY</b></li>
-          <li class="dev"><b>Back End Dev</b></li>
-          <li class="dev"><b>2015</b></li>
-        </ul>
-        <p class="presentation3">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </p>
-        <div class="languages">
-          <ul>
-            <li class="lang">HTML</li>
-            <li class="lang">CSS</li>
-            <li class="lang">JavaScript</li>
-          </ul>
-        </div>
-        <button class="button">See project</button>
-      </div>
-    </div>
+   
   </section>
   <section class="About" id="about">
```

Comment:
- [ ] I would suggest that you only wrap the main title in a `<h1>` tag. Every other title can be wrapped in other `<hx> ` tags except the `<h1>` to make your code align with HTML best practices and for semantic reasons.